api/src/main/resources/workbench-api.yaml

```diff
@@ -5546,27 +5546,16 @@ definitions:
         description: The nonce returned from the account creation API.
   SendBillingSetupEmailRequest:
     type: object
-    required:
-      - paymentMethod
     properties:
       phone:
         type: string
         description: User contact phone
       institution:
         type: string
         description: The insitution of the user.
-      paymentMethod:
```

Comment:
I'm surprised there are no backend changes - were the email templates already updated to account for this?

Reply:
That is updated in https://github.com/all-of-us/workbench/pull/5599